test/blocks/tabs/tabs.test.js

```diff
@@ -35,4 +42,58 @@ describe('tabs', () => {
     await sendKeys({ press: 'Space' });
     expect(unSelectedBtn2.ariaSelected).to.equal('false');
   });
+
+  it('paddles are disabled when tabList is not scrollable', async () => {
```

Comment:
Tests should start with a verb and make sense as a sentence
```suggestion
  it('disables paddles when tabList is not scrollable', async () => {
```

Reply:
Thanks